task/syft/0.1/tests/run-source-dir.yaml

```
@@ -0,0 +1,56 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-syft-test-source-dir
+spec:
+  workspaces:
+    - name: pipeline-pvc
+      optional: false
+  params:
+    - name: ARGS
+      type: array
+      description: The Arguments to be passed to Grype command for config image.
+  description: Test Pipeline for Grype
+  tasks:
+    - name: fetch-source
+      taskRef:
+        name: git-clone
+        kind: Task
+      workspaces:
+        - name: output
+          workspace: pipeline-pvc
+      params:
+        - name: url
+          value: https://github.com/GoTurkiye/hello-world-cli
```

Since this is a filesystem scan I think it'd be better scanning either the tekton repo itself

```suggestion
          value: https://github.com/tektoncd/catalog
```

or the nocode repo used in other tests

```suggestion
          value: https://github.com/kelseyhightower/nocode
```

rather than depending on another repo that can change underneath us

---

thanks